src/components/Form/Widgets/SelectWidget.tsx

```diff
@@ -38,6 +35,20 @@ export const SelectWidget = ({
 		};
 	});
 
+	// 	This components needs to handle two cases, single selection and multiple selections.
```

Comment:
Why does it need to handle `multiple`? We do not pass the `multiple` prop to the Autocomplete
Do you mean RJSF uses SelectWidget to handle multiple too? If so then we need to find how to know that we are handling multiple instead of single and adjust accordingly

Reply:
@myarmolinsky as explained in the comment, I mean that this case exist and the typing is considering it. I've pasted the typing to have a better view of what I mean. Todate we don't use it but we might want to use it at some point. Let's not limit the component.

Reply:
> Let's not limit the component.

100% let's not limit it, just wasn't clear on whether the support was supposed to already be present or if we were introducing it here. Understood now👌